lib/tests/2wp.js

```diff
@@ -902,6 +910,130 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should create multiple pegouts with mixed values same and above minimum', async () => {
+
+      // Arrange
+      
+      const senderRecipientInfo1 = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      await fundRskAccountThroughAPegin(rskTxHelper, btcTxHelper, senderRecipientInfo1.btcSenderAddressInfo);
+      
+      const senderRecipientInfo2 = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      await fundRskAccountThroughAPegin(rskTxHelper, btcTxHelper, senderRecipientInfo2.btcSenderAddressInfo);
+
+      const senderRecipientInfo3 = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      await fundRskAccountThroughAPegin(rskTxHelper, btcTxHelper, senderRecipientInfo3.btcSenderAddressInfo);
```

Comment:
I think it would be better to add all the recipients into an array, then iterate that array for each operation and the same for the assertions.